openmc/mgxs/library.py

```diff
@@ -1167,6 +1167,16 @@ def get_xsdata(self, domain, xsdata_name, nuclide='total', xs_type='macro',
                                     np.subtract(xsdata._total[i], np.sum(np.sum(
                                         xsdata._scatter_matrix[i][:, :, :, :, :],
                                         axis=4), axis=3))
+            # if only scatter matrices have been tallied, multiplicity cannot
+            # be accounted for
+            else:
+                msg = 'Scatter multiplicity (such as (n,xn) reactions) ' \
+                      'are ignored since multiplicity or nu-scatter matrices '\
+                      'were not tallied for '+xsdata_name
```

Comment:
Can you add a space around the '+' operator?

Reply:
done